src/App.js

```diff
@@ -12,36 +12,55 @@ import LogIn from './screens/logIn';
 import SignUp from './screens/signUp';
 
 import Loading from './components/loading';
-import LogOutButton from './components/logOutButton'
+import LogOutButton from './components/logOutButton';
+
+// ! DELETE AFTER DEBUGGING SESSION
```

Comment:
We finish right? Therefore, this comments shouldn't be here.

Reply:
I'll remove it today